docs/operating-eck/troubleshooting/common-problems.asciidoc

```diff
@@ -35,7 +35,9 @@ This error is usually an indication of a problem communicating with the validati
 [id="{p}-{page_id}-owner-refs"]
 == Copying secrets with Owner References
 
-Copying the Elasticsearch Secrets generated by ECK (for instance, the certificate authority or the elastic user) into another namespace wholesale can trigger a link:https://github.com/kubernetes/kubernetes/issues/65200[Kubernetes bug] which can delete all of the Elasticsearch-related resources (for instance, the data volumes). To avoid this, when copying Secrets between namespaces, remove the `metadata.ownerReferences` section. For example, a source secret might be:
+Since ECK 1.3.1, `OwnerReference` was removed from Elasticsearch Secrets. Copying the Elasticsearch Secrets generated by ECK (for instance, the certificate authority or the elastic user) into another namespace wholesale can trigger a link:https://github.com/kubernetes/kubernetes/issues/65200[Kubernetes bug] which can delete all of the Elasticsearch-related resources (for instance, the data volumes). Make sure you remove the `metadata.ownerReferences` section, even from Elasticsearch Secrets that were previously copied.
```

Comment:
Maybe we should clarify that the `OwnerReference` is automatically removed from a subset of those Secrets, for example:

> Since ECK 1.3.1, `OwnerReference` was removed from Secrets which are likely to be copied (the Secrets which contain the public certificates and the Secret which holds the `elastic` user credentials)

I think I would also move that part after `Copying the Elasticsearch Secrets...` and mention that if some Secrets have been copied in other namespaces before `1.3.1`, then users should remove the `OwnerReference` manually because they might still be affected, even if ECK has been upgraded.